mix.exs

```diff
@@ -90,56 +90,56 @@ defmodule RIG.MixProject do
   defp deps do
     [
       # Test coverage reporting:
-      {:excoveralls, "~> 0.10", only: :test, runtime: false},
+      {:excoveralls, "~> 0.12", only: :test, runtime: false},
       # Linting:
-      {:credo, "~> 1.0", only: [:dev, :test], runtime: false},
+      {:credo, "~> 1.3", only: [:dev, :test], runtime: false},
       # Static type checks:
       {:dialyxir, "~> 1.0.0-rc.6", only: [:dev], runtime: false},
       # OTP releases:
       {:distillery, "~> 2.1", runtime: false},
       # Documentation generator:
-      {:ex_doc, "~> 0.16", only: :dev, runtime: false},
+      {:ex_doc, "~> 0.21", only: :dev, runtime: false},
```

Comment:
would it make sense, for `excoveralls`, `credo`, `dialyxir` and `ex_doc` to use `>=` instead of `~>`?

Reply:
Dunno, tried to stick with what we have. What is your opinion?

Reply:
I guess it'd make sense cause any newer version of those tools is better than their older versions

Reply:
Ok, I'll check on Friday.